spec.bs

```diff
@@ -212,6 +218,10 @@ an issuing bank.
     credential is created and returned to the issuing bank who stores
     it in their server-side database associated with the user and
     payment instrument(s).
+    
+**TODO**: Define an Agent specific display for Enrollment, indicating that that Payment transactions will be supported (or that this is exclusively for payments)? 
```

Comment:
Assuming 'Agent specific display' here means 'a browser UX', we (Chrome) oppose this currently. We are happy that the WebAuthn UX that would be shown is sufficient understanding for the user, and we believe that the RP is best suited to build and show any more specific UX that they wish to show regarding payments.

Reply:
Ok. Let's then just state that (see my previous comment).